files/pull-sandbox-image.sh

```diff
@@ -8,4 +8,26 @@ if [[ "$(sudo ctr --namespace k8s.io image ls | grep $sandbox_image)" != "" ]];
   exit 0
 fi
 
-/etc/eks/containerd/pull-image.sh "${sandbox_image}"
```

Comment:
Any reason not to modify the `pull-image.sh` script itself to use crictl instead of ctr?

Reply:
We use `ctr content fetch` in that script because it skips image unpacking (an important optimization for the image caching feature's disk usage).